src/map/findNextWayPoint.ts

```diff
@@ -0,0 +1,67 @@
+import { Coordinate } from '@/stores/QueryStore'
+
+/**
+ * Finds the way-point that follows the part of a route that is closest to a given location
+ *
+ * @param routes one or more routes to be considered
+ * @param location the location for which the closest route should be found
+ */
+export function findNextWayPoint(
```

Comment:
Can we have a utility class and also export calcDist? (I need/have this for the navi branch already)

Reply:
Sure, will do.